src/components/ResumeBuilder/resume-builder.js

```diff
@@ -1,6 +1,9 @@
+/* NOTE: This file name was renamed to resume-builder to make the path name 
+read better when linked from the Nav component */
+
 import React from 'react'
 import { useRouter } from 'next/router'
-import '../css/LandingPage.css'
+import '../../assets/css/resumebuilder.css'
```

Comment:
+1 on the import

Reply:
The import is not needed.

Reply:
Same as https://github.com/Vets-Who-Code/vets-who-code-app/pull/421#discussion_r950646577